app/assets/stylesheets/frontend/views/_history-people.scss

```diff
@@ -1,62 +1,11 @@
-.historic-people-index {
-  .clear-person {
-    clear: both;
+.historic-people-index__section-row-header {
+  @include media(tablet) {
+    margin-top: govuk-spacing(2);
+    margin-bottom: govuk-spacing(4);
   }
 
-  .featured-profiles {
-    margin-bottom: $gutter;
-
-    .profiles {
-      @include heading-36;
-      border-bottom: $gutter-one-sixth solid govuk-colour("black");
-      margin: 0 $gutter-half $gutter-half;
-    }
-
-    .name {
-      @include core-19;
-      font-weight: bold;
-    }
-
-    .term {
-      color: $govuk-secondary-text-colour;
-      display: block;
-    }
-
-    .person-excerpt {
-      .inner {
-        .image-holder {
-          @include govuk-media-query($until: tablet) {
-            margin-right: govuk-spacing(2);
-          }
-
-          img {
-            max-width: $full-width;
-          }
-        }
-      }
-    }
-
-    @include media(tablet) {
-      margin-bottom: $gutter * 2;
-    }
-  }
-
-  .foreign-secretaries {
-    .heading {
-      h3 {
-        @include heading-36;
-        font-weight: bold;
-        padding: 0 $gutter-half;
-
-        span {
-          @include core-19;
-          display: block;
-        }
-        @include media(tablet) {
-          padding-top: 0;
-        }
-      }
-    }
+  .gem-c-image-card__description {
```

Comment:
This override of the image card from the gem isn't ideal - is there a way it could be part of the component instead?

Reply:
I toyed with this. I wondered how many other `image_card`s across the site would need the description colour to be changed - to the point where it would be re-used more than just here.   Especially since we discourage image use.  

I did wonder if we had a colour override so I could use something like `govuk-!-colour-grey`

🤔  With all that said: this breaks conventions - let me extend the `image_card` 👍🏼